alphagov/paas-team-manual
# [SUPPORT] CF UAA user management guide

## What

Document how to do some things with `uaac`. This came out of discussion in alphagov/paas-cf#925

## How to review

1. Check that the words make sense
1. Try the commands

## Who can review

Ideally @paroxp, but anyone can.
